packages/init/bootstrap-starter/near-runner/__tests__/main.ava.ts

```diff
@@ -0,0 +1,162 @@
+/**
+ * Welcome to near-runner-ava!
+ *
+ * This is a working test which checks the functionality of [the status-message
+ * contract][1]. For quick reference, here's the contract's implementation:
+ *
+ *     impl StatusMessage {
+ *         pub fn set_status(&mut self, message: String) {
+ *             let account_id = env::signer_account_id();
+ *             self.records.insert(&account_id, &message);
+ *         }
+
+ *         pub fn get_status(&self, account_id: String) -> Option<String> {
+ *             return self.records.get(&account_id);
+ *         }
+ *     }
+ *
+ * As you can see, this contract only has two methods, a setter and a getter.
+ * The setter sets a free-form status message for the account that called the
+ * contract. The getter accepts an `account_id` param and returns the status for
+ * that account.
+ *
+ * The tests below create a local blockchain with this contract deployed to
+ * one account and two more accounts which store statuses in the contract.
+ *
+ *   [1]: https://github.com/near-examples/rust-status-message/tree/4e4767db257b748950bb3393352e2fff6c8e9b17
+ */
+
+/**
+ * Start off by importing Runner from near-runner-ava. This is usually the only
+ * import you'll need.
+ */
+import {Runner} from 'near-runner-ava';
+
+/**
+ * Create a new runner. In local sandbox mode, this will:
+ *
+ *   - Create a new local blockchain
+ *   - Create the root account for that blockchain (see `root` below)
+ *   - Execute any actions passed to the function
+ *   - Shut down the newly created blockchain, but *save the data*
+ */
+const runner = Runner.create(async ({root}) => {
+  // Create a subaccount of the root account, like `alice.sandbox`
+  // (the actual account name is not guaranteed; you can get it with `alice.accountId`)
+  const alice = await root.createAccount('alice');
+
+  // Create a subaccount of the root account, and also deploy a contract to it
+  const contract = await root.createAndDeploy(
+    // Subaccount name
+    'status-message',
+
+    // Relative path (from package.json location) to the compiled contract file
+    // which will be deployed to this account
+    'compiled-contracts/status_message.wasm',
+
+    // Provide `method` and `args` to call in the same block as the deploy
```

Comment:
```suggestion
    // Provide `method` and `args` to call in the same transaction as the deploy
```

Not sure if has to be the same block, but I could be wrong.

Reply:
Seems like it must happen in the same block if it happens in the same transaction. Thanks for the precision!